contrib/pg_tde/documentation/docs/functions.md

```diff
@@ -234,7 +248,7 @@ The `ensure_new_key` parameter instructs the function how to handle a principal
 
 Creates or rotates the default principal key for the server using the specified key provider.
 
-The default key is automatically used as a principal key by any database that doesn't have a specific principal key created the first time an encrypted database object is created.
+The default key is automatically used as a principal key by any database that doesn't have a specific principal key already created when the first encrypted database object is created.
```

Comment:
```suggestion
The default key is automatically used as a principal key by any database that doesn't have an individual key provider and key configuration.
```

Reply:
This part: already created when the first encrypted database object is created. is super confusing. Can we get rid of it without losing the sense?

Reply:
Good comments but this is a merge commit so not the place to fix this. Sorry! Let's fix all of these as a separate PR.